packages/api-providers/src/polkadot/vanchor-deposit.ts

```diff
@@ -38,7 +38,7 @@ const logger = LoggerService.get('PolkadotVBridgeDeposit');
 // TODO: export this from webb.js
 
 // The Deposit Payload is the note and [treeId]
```

Comment:
Can we update the comment here for `wrappableAssetAddress?` added as the second parameter? Is it worthwhile to narrow the type to just `Number | undefined` as the depositPayload should always be for the current chain's Currency target (treeId).